appveyor.yml

```diff
@@ -0,0 +1,21 @@
+stack: jdk 8
+environment:
+  matrix:
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
```

Comment:
It is better to use images like here:
https://github.com/jfrog/jfrog-idea-plugin/blob/master/appveyor.yml#L1

Reply:
We will leave it that way to avoid setting a redundant env_var on the linux machine